process/peer/process.go

```diff
@@ -312,44 +312,66 @@ func (p *validatorStatistics) checkForMissedBlocks(
 		return nil
 	}
 
+	tm := core.NewTimeMeasure()
+	tm.Start("checkForMissedBlocks")
+	defer func() {
+		tm.Finish("checkForMissedBlocks")
+		log.Debug("measurements checkForMissedBlocks", tm.GetMeasurements()...)
+	}()
+
 	for i := previousHeaderRound + 1; i < currentHeaderRound; i++ {
-		consensusGroup, err := p.nodesCoordinator.ComputeValidatorsGroup(prevRandSeed, i, shardId)
+		tmInner := core.NewTimeMeasure()
```

Comment:
Should be analyzed which of the next measurements really need to be done.

Reply:
I would leave them all as to have a clearer picture the next time we hit a problem in this area

Reply:
ok